packages/legend-extension-dsl-data-space/src/stores/query/DataSpaceQueryBuilderState.ts

```diff
@@ -152,25 +162,27 @@ export class DataSpaceQueryBuilderState extends QueryBuilderState {
   }
 
   showAdvancedSearchPanel(): void {
-    this.advancedSearchState = new DataSpaceAdvancedSearchState(
-      this.applicationStore,
-      this.graphManagerState,
-      this.depotServerClient,
-      {
-        viewProject: this.viewProject,
-        viewSDLCProject: this.viewSDLCProject,
-      },
-      {
-        groupId: this.groupId,
-        artifactId: this.artifactId,
-        versionId: this.versionId,
-        title: this.dataSpace.title,
-        name: this.dataSpace.name,
-        path: this.dataSpace.path,
-        defaultExecutionContext: this.dataSpace.defaultExecutionContext.name,
-      },
-      this.versionId === SNAPSHOT_VERSION_ALIAS,
-    );
+    if (this.projectInfo) {
```

Comment:
shouldnt we be using the flag above for this as well. `isAdvancedDataSpaceSearchEnabled`